catalogs/qhimm.xml

```diff
@@ -611,16 +611,16 @@ Not compatible with other Gameplay mods.</Description>
       <Link>https://www.nexusmods.com/finalfantasy7/mods/82</Link>
       <LatestVersion>
 		<Link>iros://GDrive/1sIwpkJOWG5RpdTvoFXUFxAdcg_AVB8mW</Link>
-        <Version>1.23</Version>
-        <ReleaseDate>2023-10-22</ReleaseDate>
+        <Version>1.24</Version>
+        <ReleaseDate>2024-1-2</ReleaseDate>
```

Comment:
Blows up for me with `There is an error in XML document (616, 10)`, I'm thinking the date format might need zero-padding to match the others, so:
```suggestion
        <ReleaseDate>2024-01-02</ReleaseDate>
```
Unable to test this though at the moment as I'm not set up to run a build locally, but I manually add the previous version as a source (`iros://Url/https$raw.githubusercontent.com/tsunamods-codes/7th-Heaven-Catalogs/0eb95aab1b46a59681820dd367effcf8d89e1e51/catalogs/qhimm.xml`) it works again.